src/table/interfaces.tsx

```diff
@@ -281,17 +281,14 @@ export interface TableProps<T = any, V = any> extends BaseComponentProps {
 export namespace TableProps {
   export type TrackBy<T> = string | ((item: T) => string);
 
-  export type EditableColumnDefinition<T, V = any> = Extract<ColumnDefinition<T, V>, ColumnEditConfig<T, V>>;
+  export type EditableColumnDefinition<T> = Extract<ColumnDefinition<T>, ColumnEditConfig<T>>;
```

Comment:
Is this one used anywhere in public API? If not, should be removed from `TableProps` namespace

Reply:
Its used internally, I'll removing it from the namespace; maybe even completely